invenio_collections/models.py

```diff
@@ -287,6 +318,12 @@ def ancestors(self):
             output |= c.dad.ancestors
         return output
 
+    def is_root(self):
```

Comment:
I guess this definition of the root collection could lead to troubles.  

- It is true that there should be only one collection named CFG_SITE_NAME and that its collection ID should be 1.  This defines our "home collection".  

- However, you can have several "floating home collections" that people don't want to attach in the regular collection tree descending from the home collection.  E.g. imagine a "Drafts" collection that is restricted and searchable only when people use in the UI `cc=Drafts`.  Otherwise this collection wouldn't pop up in the facets or anywhere else, not being a descendant of the home collection.  

Using this definition of `is_root()`, we'd get True for both the Home collection and the Drafts collection, leading to name validation troubles.  The fact that a collection does not have any ancestors is not enough to make it the root node...